feature-libs/product-configurator/rulebased/components/attribute/product-card/configurator-attribute-product-card.component.ts

```diff
@@ -10,40 +10,64 @@ import { Configurator } from '../../../core/model/configurator.model';
 import { FormControl } from '@angular/forms';
 import { Subscription } from 'rxjs';
 
+interface QuantityUpdateEvent {
+  quantity: number;
+  valueCode: string;
+}
+
 @Component({
   selector: 'cx-configurator-attribute-product-card',
   templateUrl: './configurator-attribute-product-card.component.html',
 })
 export class ConfiguratorAttributeProductCardComponent
   implements OnDestroy, OnInit {
   quantity = new FormControl(1);
+  loading = false;
```

Comment:
maybe `BehaviorSubject()` instead of variable? does it work correctly this way?

Reply:
it is working, but yes, BehaviorSubject is better solution, changed to BehaviorSubject